libsqlite3-sys/src/lib.rs

```diff
@@ -50,6 +50,26 @@ pub enum Limit {
     SQLITE_LIMIT_WORKER_THREADS = 11,
 }
 
+#[cfg(feature = "loadable_extension_embedded")]
+// bindings were built with loadable_extension_embedded:
+// define sqlite3_api as an extern since this code will be embedded
+// within a loadable extension that defines and exports this itself
+extern "C" {
+    #[no_mangle]
+    pub static mut sqlite3_api: *mut sqlite3_api_routines;
```

Comment:
This should probably be an AtomicPtr, and have a Release store and Acquire loads. Otherwise it's a data race.

Reply:
Discussion moved to https://github.com/rusqlite/rusqlite/pull/910/files#r765305470